src/Microsoft.Identity.Web/Resource/RegisterValidAudience.cs

```diff
@@ -57,11 +58,12 @@ internal class RegisterValidAudience
             SecurityToken securityToken,
             TokenValidationParameters validationParameters)
         {
-            JwtSecurityToken? token = securityToken as JwtSecurityToken;
-            if (token == null)
+            var claims = securityToken switch
```

Comment:
@brentschmaltz @jennyf19 
What about having an extension method in Wilson to get the claims from a SecurityToken?

Reply:
I think that makes sense.